### ISSUE TITLE
canvas anti-alias and Chrome

### ISSUE THREAD

**Issue by caasi** [ChineseCubes/react-odp#3]
@Rex-How mentioned this problem in: https://trello.com/c/t6pYB54h today.

But Chrome do no anti-alias when clipping: https://code.google.com/p/chromium/issues/detail?id=7508.

**Comment by audreyt** [comment]
How about draw at 2x the size or 3x and CSS-transform back?

**Comment by audreyt** [comment]
(CSS 2d scale would probably work without the need for 3d transform.)

**Comment by caasi** [comment]
Yes, I remember your css hack, but I forget if I have implemented it in sprite-stroke.ls or not. :-p
